src/MoBi.Presentation/MoBi.Presentation.csproj

```diff
@@ -28,6 +28,9 @@
     <PackageReference Include="Northwoods.GoWin" Version="5.2.0" />
     <PackageReference Include="OSPSuite.TeXReporting" Version="3.0.0.4" />
     <PackageReference Include="OSPSuite.Utility" Version="4.0.0.4" />
+    <PackageReference Include="OSPSuite.Presentation.Importer" Version="10.0.92" />
+    <PackageReference Include="OSPSuite.TeXReporting" Version="3.0.0.4" />
+    <PackageReference Include="OSPSuite.Utility" Version="4.0.0.4" />
```

Comment:
we have multiple of the same again. WTF

Reply:
this is weird....haven't seen this one before...